lib/l10n/app_en.arb

```diff
@@ -1,301 +1,430 @@
+    "q_have_account_info": "Have account info?",
+    "l_run_diagnostics": "Run diagnostics",
+    "l_log_level": "Log level: {level}",
+    "@l_log_level": {
         "placeholders": {
-            "used": {},
-            "capacity": {}
+            "level": {}
         }
     },
-    "oath_custom_icons": "Custom icons",
-    "oath_custom_icons_description": "Icon packs can make your accounts more easily distinguishable with familiar logos and colors.",
-    "oath_custom_icons_replace": "Replace icon pack",
-    "oath_custom_icons_loading": "Loading icon pack\u2026",
-    "oath_custom_icons_load": "Load icon pack",
-    "oath_custom_icons_remove": "Remove icon pack",
-    "oath_custom_icons_icon_pack_removed": "Icon pack removed",
-    "oath_custom_icons_err_icon_pack_remove": "Error removing icon pack",
-    "oath_custom_icons_choose_icon_pack": "Choose icon pack",
-    "oath_custom_icons_icon_pack_imported": "Icon pack imported",
-    "oath_custom_icons_err_icon_pack_import": "Error importing icon pack: {message}",
-    "@oath_custom_icons_err_icon_pack_import": {
+    "l_character_count": "Character count",
+    "l_learn_more": "Learn\u00a0more",
+
+    "@_theme": {},
+    "l_app_theme": "App theme",
+    "l_choose_app_theme": "Choose app theme",
+    "l_system_default": "System default",
+    "l_light_mode": "Light mode",
+    "l_dark_mode": "Dark mode",
+
+    "@_yubikey_selection": {},
+    "l_select_to_scan": "Select to scan",
+    "l_hide_device": "Hide device",
+    "l_show_hidden_devices": "Show hidden devices",
+    "l_sn_serial": "S/N: {serial}",
+    "@l_sn_serial" : {
         "placeholders": {
-            "message": {}
+            "serial": {}
+        }
+    },
+    "l_fw_version": "F/W: {version}",
+    "@l_fw_version" : {
+        "placeholders": {
+            "version": {}
         }
     },
-    "oath_custom_icons_learn_more": "Learn\u00a0more",
-    "oath_custom_icons_err_import_general": "Import error",
-    "oath_custom_icons_err_file_not_found": "File not found",
-    "oath_custom_icons_err_file_too_big": "File size too big",
-    "oath_custom_icons_err_invalid_icon_pack": "Invalid icon pack",
-    "oath_custom_icons_err_filesystem_error": "File system operation error",
 
-    "widgets_cancel": "Cancel",
-    "widgets_close": "Close",
+    "@_yubikey_interactions": {},
+    "l_insert_yk": "Insert your YubiKey",
+    "l_insert_or_tap_yk": "Insert or tap a YubiKey",
+    "l_unplug_yk": "Unplug your YubiKey",
+    "l_reinsert_yk": "Reinsert your YubiKey",
+    "l_place_on_nfc_reader": "Place your YubiKey on the NFC reader",
+    "l_replace_yk_on_reader": "Place your YubiKey back on the reader",
+    "l_remove_yk_from_reader": "Remove your YubiKey from the NFC reader",
+    "l_try_reinsert_yk": "Try to remove and reinsert your YubiKey.",
+    "l_touch_required": "Touch Required",
+    "l_touch_button_now": "Touch your YubiKey now",
+    "l_touch_button_now": "Touch the button on your YubiKey now",
```

Comment:
it looks like this key is twice here

Reply:
On Android we originally used `Touch the button on your YubiKey now.`

Reply:
Good catch! We can probably choose one and use it for both platforms.